zetaclient/chains/evm/observer/v2_inbound.go

```diff
@@ -104,6 +107,7 @@ func (ob *Observer) observeGatewayDeposit(
 			}
 		}
 
+		metrics.InboundObservationsBlockScanTotal.WithLabelValues(ob.Chain().Name).Inc()
```

Comment:
The vote could be skipped by a few downstream logic, for example: voted or not, ballot exists or not. Most of the times, accumulated number in here will be much more than actual # of votes posted. Moving counting to downstream produces more accurate result.

Reply:
yeah that makes sense 

However this metric is for blockscanned , it should just increase whenever zetaclient sees a new event 
the actual vote posted is defined by 
InboundVotesPostedWith500KGasLimitTotal
and 
InboundVotesPostedWith7MGasLimitTotal

having a lot of difference between the two means that validation failed in some step